<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMM_ENT_DETAYLARI_BATCH</class>
 <widget class="QDialog" name="SMM_ENT_DETAYLARI_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,0" columnstretch="1,0,0,0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="label_sirket">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Kaydet --&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_sirket">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" colspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>170</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_muh_fis_no">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Muhasebe Fiş No</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCommaEdit" name="commaEdit_muh_fis_no">
     <property name="whatsThis">
      <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
     </property>
     <property name="QCommaEdit" stdset="0">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_cari_fis_no">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cari Fiş No</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCommaEdit" name="commaEdit_cari_fis_no">
     <property name="whatsThis">
      <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
     </property>
     <property name="QCommaEdit" stdset="0">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_banka_fis_no">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Banka Fiş No</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCommaEdit" name="commaEdit_bnk_fis_no">
     <property name="whatsThis">
      <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
     </property>
     <property name="QCommaEdit" stdset="0">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Kasa Hesabı</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSearchEdit" name="searchEdit_muh_hesap_kodu" native="true">
     <property name="whatsThis">
      <string>Arama widget. </string>
     </property>
     <property name="QSearchEdit" stdset="0">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_muh_hesap_ismi"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_tahsil_edildi_odendi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBox_tevkifatli_mi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Tevkifatlı  mı ?</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QFrame" name="frame_tevkifat_detaylari">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,5,10,10">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tevkifat Yüzdesi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCommaEdit" name="commaEdit_tevkifat_yuzdesi">
        <property name="whatsThis">
         <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
        </property>
        <property name="QCommaEdit" stdset="0">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>106</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>268</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>404</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="5">
    <widget class="QWidget" name="widget_batch_buttons" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCommaEdit</class>
   <extends>QLineEdit</extends>
   <header>QCommaEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QSearchEdit</class>
   <extends>QWidget</extends>
   <header>QSearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
